website/docs/patterns/filter-patterns/partials/guidelines/guidelines.md

```diff
@@ -171,30 +171,215 @@ If applying filters to a data set yields no results, offer users an option to cl
 
-![Multiple filter bar rows](/assets/patterns/filter-patterns/overflow-filter-bar-multiple-rows.png =559x*)
+<Doc::ImageCaption @text="Allowing applied filters to wrap and reflow naturally." />
 
-!!!
+#### Intermediate example
+
+Filtering a complex data set may result in numerous filters overwhelming the UI and detracting from the Table's content.
+
+In this scenario, consider moving the applied filter [Tags](/components/tag) to a `small`, `card` [Accordion](/components/accordion) using the `containsInteractive` variant. This pattern:
+
+- Accommodates numerous applied filters
+- Simplifies the UI, reducing cognitive load
+- Supports complex, interactive content within the Accordion (e.g., a "Clear all filters" function)
+
+![Intermediate example of applied filters within an open Accordion](/assets/patterns/filter-patterns/applied-filters-overflow-intermediate-example-open.png)
+
+<Doc::ImageCaption @text="Moving the applied filters to an Accordion." />
+
+### Filter parameters overflow
+
+In complex data sets with numerous filterable parameters, considering establishing a hierarchy of filters by:
+
+1. Prioritizing crucial filters directly in the UI and associating them directly with the Table and data set
+2. Moving less important filter parameters to a [Flyout](/components/flyout), using [Checkbox groups](/components/form/checkbox) for each parameter and its values.
+
+This approach offers a scalable solution for highly complex data sets while communicating a natural hierarchy of filter importance.
+
+![Trigger the overflow of filter parameters with a Button](/assets/patterns/filter-patterns/filter-parameters-overflow-more-filters.png)
 
-### Sidebar overflow
+<Doc::ImageCaption @text="Use a secondary Button to trigger a Flyout." />
 
-Given that the page-level sidebar occupies the height of the viewport, take filters within a sidebar should not overflow into an interstitial component like a [Flyout](/components/flyout). Instead, utilize the height of the viewport and the consider introducing a scrollable area within the sidebar if many filters are necessary.
+![Display the overflow of filter parameters within a Flyout](/assets/patterns/filter-patterns/filter-parameters-overflow-flyout.png)
+
+<Doc::ImageCaption @text="Display the overflow of filter parameters in a Flyout" />
+
+#### Accordions within a Flyout
+
+For excessive filter parameters causing a long scroll in the [Flyout](/components/flyout), consider placing each parameter in a `flush`, `small` [Accordion](/components/accordion), and optionally, add an "Expand/Collapse All" Button. This approach gives users more control over the UI and allows for visual comparison between filter parameters.
+
+![Accordions within a Flyout containing filter parameters](/assets/patterns/filter-patterns/filter-parameters-overflow-flyout-accordion.png)
+
+<Doc::ImageCaption @text="Use Accordions within a Flyout to support an excessive number of filter parameters." />
+
+### Filter hierarchy
```

Comment:
[suggestion non-blocking] This section feels like it re-iterates lines 212-213. Could it be combined with that content somehow?

Reply:
I think I'll just kill this whole section since it's referenced in other places